docker-compose.yml

```diff
@@ -0,0 +1,175 @@
+version: "3.7"
+services:
+  proxy:
+    image: bahmniindiadistro/bahmni-proxy-abdm:${PROXY_IMAGE_TAG:?}
+    volumes: 
+    #   - ${CERTIFICATE_PATH}:/etc/tls
+    - "bahmni-patient-images:/home/bahmni/patient_images"
+    - "bahmni-document-images:/home/bahmni/document_images"
+
+    ports:
+      - "80:80"
+      - "443:443"
+
+  openmrs:
+    profiles: ["default","openmrs"]
+    image: bahmniindiadistro/bahmni-openmrs-abdm:${OPENMRS_IMAGE_TAG:?}
+    environment:
+      DB_DATABASE: ${OPENMRS_DB_NAME:?}
+      DB_HOST: ${OPENMRS_DB_HOST:?}
+      DB_USERNAME: ${OPENMRS_DB_USERNAME:?}
+      DB_PASSWORD: ${OPENMRS_DB_PASSWORD:?}
+      DB_CREATE_TABLES: ${OPENMRS_DB_CREATE_TABLES}
+      DB_AUTO_UPDATE: ${OPENMRS_DB_AUTO_UPDATE}
+      MODULE_WEB_ADMIN: ${OPENMRS_MODULE_WEB_ADMIN}
+      DEBUG: ${OPENMRS_DEBUG}
+      OPENELIS_HOST: ${OPENELIS_HOST:?}
+      OPENELIS_PORT: ${OPENELIS_PORT:?}
+      OPENELIS_ATOMFEED_USER: ${OPENELIS_ATOMFEED_USER:?}
+      OPENELIS_ATOMFEED_PASSWORD: ${OPENELIS_ATOMFEED_PASSWORD:?}
+    ports:
+      - "8080:8080"
+    volumes:
+      #- "openmrs-data:/usr/local/tomcat/.OpenMRS/modules"
+      #- "${BAHMNI_OPENMRS_MODULES_PATH:?}/:/usr/local/tomcat/.OpenMRS/modules/"
+      #- "${BAHMNI_CONFIG_PATH:?}/:/etc/bahmni_config/"
+      - "bahmni-patient-images:/home/bahmni/patient_images"
+      - "bahmni-document-images:/home/bahmni/document_images"
+    depends_on:
+      - openmrsdb
+
+  openmrsdb:
+    image: bahmni/openmrs-db:${OPENMRS_DB_IMAGE_TAG:?}
+    restart: always
+    profiles: ["default","openmrs"]
+    environment:
+      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:?}
+      MYSQL_DATABASE: ${OPENMRS_DB_NAME:?}
+      MYSQL_USER: ${OPENMRS_DB_USERNAME:?}
+      MYSQL_PASSWORD: ${OPENMRS_DB_PASSWORD:?}
+    ports:
+      - "3306:3306"
+    volumes:
+      - "openmrsdbdata:/var/lib/mysql"
+
+  bahmni-web:
+    image: bahmniindiadistro/bahmni-web-abdm:${BAHMNI_WEB_IMAGE_TAG:?}
+    profiles: ["default","openmrs"]
+    #volumes:
+      #- "${BAHMNI_APPS_PATH:?}/ui/app/:/usr/local/apache2/htdocs/bahmni"
+      #- "${BAHMNI_APPS_PATH:?}/ui/node_modules/@bower_components/:/usr/local/apache2/htdocs/bahmni/components"
+      #- "${BAHMNI_CONFIG_PATH:?}/:/usr/local/apache2/htdocs/bahmni_config/"
+
+  db:
```

Comment:
A lot of services seem to depend on the same "db" service. Is the DB shared between HIP, HIU and OTP? Do they create separate schemas within one postgres server, or are all of them connecting to same DB (and hence tables are all created in same schema)?

Reply:
Ideally each service should be having its own dedicated DB.. and not share data across services via a common DB. Is that happening here?

Reply:
They have their separate schemas , own dedicated DB